pom.xml

```diff
@@ -19,7 +19,7 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-api</artifactId>
-            <version>5.9.1</version>
+            <version>5.9.2</version>
```

Comment:
Why did you choose version 5.9.2 and not 5.9.3?

Reply:
@levinjoller yes I saw JUnit 5.9.3 is the latest release, but it is still in the snapshot stage, so felt it is better to choose a latest "stable" version which is 5.9.2 currently !

Reply:
No specific link just googled it actually, as the application might break so did not wanted to take a chance so went ahead with the stable 5.9.2 version 👍

Reply:
> Thank you very much for your explanation. Can you share the link where you found this information. As far as I can see, 5.9.3 is the latest stable version, but I may be wrong. [Junit5 MVN repository](https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-api)

No specific link just googled it actually, as the application might break so did not wanted to take a chance with a 2 month old 5.9.3 so went ahead with the stable 5.9.2 version released  in January 2023👍

Reply:
I can change it if you want, as the link you posted shows even 5.9.3 as stable 🤔

Reply:
> I can change it if you want, as the link you posted shows even 5.9.3 as stable 🤔

Yes please. I prefer version 5.9.3 as it seems to be the latest stable version. Thank you for editing.

Reply:
Done @vishalsingh2972  made the change you requested!

Reply:
Done @levinjoller made the change you requested!